hr_timesheet_task_required/views/res_config_settings.xml

```diff
@@ -17,6 +17,7 @@ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
                     </div>
                     <div class="o_setting_right_pane">
                         <label for="is_timesheet_task_required"/>
+                        <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." role="img" aria-label="Values set here are company-specific." groups="base.group_multi_company"/>
```

Comment:
Put this only if the group multi-company is present

Reply:
Check the very end of the line - is that what needed?

Reply:
Yes! That's the problem with long lines here in GitHub, that are difficult to be seen...

Reply:
Yeah, true!